gum/backend-linux/gumandroid.c

```diff
@@ -572,11 +572,9 @@ static const GumModuleDetails *
 gum_try_init_linker_details (void)
 {
   const GumModuleDetails * result = NULL;
-  gchar * maps, ** lines, *linker_path;
+  gchar ** lines, * linker_path, * maps;
   gint num_lines, vdso_index, i;
```

Comment:
```
  gchar * maps, ** lines;
  gint num_lines, vdso_index, i;
  gchar * linker_path = NULL;
```

Reply:
This is already fixed in the latest commit (I believe, this view is getting cluttered for me)

Reply:
Ah yes sorry, my bad.